### ISSUE TITLE
Tries to load module object instead of name

### ISSUE THREAD

**Issue by TimvdLippe** [GoogleChromeLabs/worker-plugin#4]
I have the following in my `main-thread.js`:

```js
const worker = new Worker('./counter-worker.js', {type: 'module'});
```

With an empty `counter-worker.js`. When I open the build, it loads http://localhost:8000/build/[object%20Module] rather than http://localhost:8000/build/0.main-thread.worker.js It appears that the following diff in the build output makes it work again:

```diff
- const worker = new Worker(__webpack__worker__1, {type: 'module'});
+ const worker = new Worker(__webpack__worker__1.default, {type: 'module'});
```

As the console output of `__webpack__worker__1` is the following:

```js
> Module {default: "0.main-thread.worker.js", __esModule: true, Symbol(Symbol.toStringTag): "Module"}
default: "0.main-thread.worker.js"
Symbol(Symbol.toStringTag): "Module"
__esModule: true
__proto__: Object
```

That appears to be the result of `__webpack_require__(1)`, which returns the module object. However, I can't find any reference to the Module object in that module.

This happens on both Chrome 69 and Firefox 62.

**Comment by developit** [comment]
Ah - this would also vary depending on whether your Webpack configuration is set up to transpile all .js files as well.  #5 should fix.

**Comment by developit** [comment]
Fix released in 1.1.0 :)

**Comment by TimvdLippe** [comment]
Thanks!
